packages/datadog-plugin-pg/src/index.js

```diff
@@ -32,9 +32,15 @@ function createWrapQuery (tracer, config) {
       }
 
       const originalCallback = pgQuery.callback
-      const statement = pgQuery.text
+      const statement = (pgQuery.cursor && pgQuery.cursor.text) || pgQuery.text
       const params = this.connectionParameters
 
+      if (pgQuery.readable) {
+        // pgQuery is a readable stream (using a cursor)
```

Comment:
In general I prefer self-explanatory code to comments, so I'd say to create an `isCursor` function that takes the pgQuery.

Reply:
well, this is checking if it's readable stream (which would normally happen when query is a cursor) which is what we really want to ask here to know if it will emit this events, but I can make an `isReadableStream`

Reply:
(added)